ui/src/heap/HeapRow.tsx

```diff
@@ -64,7 +58,7 @@ function Actions({
   author,
 }: TopBarProps) {
   const [deleteOpen, setDeleteOpen] = useState(false);
-  const nest = useNest();
+  const chFlag = useChannelFlag();
```

Comment:
hmm curious why this would be wrong, is there an issue with the nest hook?

Reply:
Yes it's probably obsolete. It looks for an `app` router param, which I don't see defined in any routes.